%YAML:1.0
---

adjust_debug: 0
max_fitter: 900
offset: 0.11
min_range: 0.1
max_range: 1.0



